src/strategies/erc20-votes-with-override/index.ts

```diff
@@ -0,0 +1,144 @@
+import { formatUnits } from '@ethersproject/units';
+import { isAddress } from '@ethersproject/address';
+import { multicall } from '../../utils';
+
+export const author = 'serenae-fansubs';
+export const version = '0.1.0';
+
+const getVotesName = "getVotes";
+const getVotesABI = ["function getVotes(address account) view returns (uint256)"];
+const balanceOfName = "balanceOf";
+const balanceOfABI = ["function balanceOf(address account) view returns (uint256)"];
+const delegatesName = "delegates";
+const delegatesABI = ["function delegates(address account) view returns (address)"];
+
+/*
+  Counts votes from delegates, and also from individual delegators who wish
+  to override the vote of their delegate.
+
+  Makes three multicalls for votes, delegates, and balances.
+
+  If getVotes returns a nonzero value, then that is used. Then any delegators
+  that have individually voted will have their balances subtracted from the
+  result.
+
```

Comment:
How do the `getVotes` know that users have already voted? Sorry @serenae-fansubs Trying to get my brain around it :)

Reply:
@ChaituVR it's the same method used in the regular erc20-votes strategy: https://github.com/snapshot-labs/snapshot-strategies/blob/master/src/strategies/erc20-votes/index.ts#L24

Reply:
I also updated the comments there to make it more clear that it's talking about _voting power_, and refactored to make it a bit easier to follow.